Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -1071,4 +1092,173 @@ final class PledgeViewModelTests: TestCase {
       self.createBackingError.assertValues(["Something went wrong."])
     }
   }
+
+  func testUpdateBacking_Success() {
+    let reward = Reward.postcards
+
+    let project = Project.cosmicSurgery
+      |> Project.lens.state .~ .live
+      |> Project.lens.personalization.isBacking .~ true
+      |> Project.lens.personalization.backing .~ (
+        .template
+          |> Backing.lens.paymentSource .~ GraphUserCreditCard.amex
+          |> Backing.lens.status .~ .errored
```

Comment:
Any reason this status is `.errored`? Maybe `.pledged` would be more consistent with what we'd be likely to see during this flow?

Reply:
I probably copy-pasted this from somewhere 😄